tests/e2e/tests/tags.test.js

```diff
@@ -0,0 +1,124 @@
+const { TutorialBase } = require('../tutorials/tutorialBase');
+const utils = require('../utils/utils');
+const auto = require('../utils/auto');
+
+const {
+  user,
+  pass,
+} = utils.getUserAndPass();
+
+const STUDY_NAME = 'study_tag_test';
+const TAG_NAME = 'tag_tag_test';
+const TAG_NAME_2 = 'tag_tag_test_2';
+let studyId = null;
+let tagId = null;
+
+/**
+ * This function records the IDs of the study and tag created in order to later remove them.
+ */
+const responseHandler = response => {
+  if (response.url().endsWith('/tags') && response.request().method() === 'POST') {
+    response.json()
+      .then(({ data: { id } }) => {
+        console.log("Tag created, id", id);
+        tagId = id;
+      });
+  }
+  if (response.url().endsWith('/projects') && response.request().method() === 'POST') {
+    response.json()
+      .then(({ data: { uuid } }) => {
+        console.log("Study created, uuid", uuid);
+        studyId = uuid;
+      });
+  }
+}
+
+const waitAndClick = (selector, clickCount=1) => page.waitForSelector(selector)
```

Comment:
Why not using the one from tests/e2e/utils.js?

Reply:
done